Clinical-Genomics/chanjo2
# Update and fix deprecated GitHub actions

### This PR adds | fixes:
- An attempt to fix deprecated code in GitHub actions, resulting in [warnings](https://github.com/Clinical-Genomics/chanjo2/actions/runs/4664906042) - fix #55

### How to test:
- Check the actions tab and make sure that this branch doesn't trigger any warnings

### Review:
- [x] Tests executed by GitHub actions

This [version](https://semver.org/) is a:
